src/modules/recipe/helloExpressionsTrack/__tests__/helloExpressionsTrack.test.js

```diff
@@ -0,0 +1,84 @@
+import { createElement } from 'lwc';
+import HelloExpressionsTrack from 'recipe/HelloExpressionsTrack';
```

Comment:
Should be `import HelloExpressionsTrack from 'recipe/helloExpressionsTrack';`. That's also the reason why the unit test is failing.

Reply:
More background info on this case-sensitivity issue: https://stackoverflow.com/questions/13617863/can-i-force-node-js-require-to-be-case-sensitive